src/main/java/ch/iceguard/functions/Function.java

```diff
@@ -1,77 +1,60 @@
 package ch.iceguard.functions;
 
-import java.util.*;
-import java.util.function.Consumer;
-
-import com.microsoft.azure.functions.annotation.*;
 import com.microsoft.azure.functions.*;
-import com.mongodb.MongoClient;
-import com.mongodb.MongoClientURI;
-import com.mongodb.client.MongoCollection;
-import com.mongodb.client.MongoDatabase;
-import org.bson.Document;
+import com.microsoft.azure.functions.annotation.AuthorizationLevel;
+import com.microsoft.azure.functions.annotation.FunctionName;
+import com.microsoft.azure.functions.annotation.HttpTrigger;
 
-import static com.mongodb.client.model.Filters.eq;
+import java.util.Arrays;
```

Comment:
Nice, you implemented the DAO pattern. Therefore you can write tests now. Bravo! :)